src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -1133,7 +1170,18 @@ const formConfig = {
               get(
                 'applicantRelationshipToSponsor.relationshipToVeteran',
                 formData?.applicants?.[index],
-              ) === 'spouse'
+              ) === 'spouse' &&
+              ((get('sponsorIsDeceased', formData) &&
+                [
+                  'marriedTillDeathNoRemarriage',
+                  'marriedTillDeathRemarriedAfter55',
+                ].includes(
+                  get(
+                    'applicantSponsorMarriageDetails.relationshipToVeteran',
+                    formData?.applicants?.[index],
+                  ),
+                )) ||
+                !get('sponsorIsDeceased', formData))
```

Comment:
We show this page IF:
- The applicant is currently married to the living sponsor
OR
- The applicant was married to the sponsor till the sponsor died.